packages/api/user/application/services/finder.ts

```diff
@@ -0,0 +1,11 @@
+import { Result } from "neverthrow"
+import { NotFoundUser } from "~/user/domain/exceptions/not-found"
+import UserId from "~/user/domain/models/id"
+import { UserDto } from "~/user/dto/response/user"
+
+export type UsersFinder = {
+  find(id: UserId): Promise<Result<UserDto, NotFoundUser>>
+  getAll(): Promise<UserDto[]>
```

Comment:
This method is no used at the moment. You can leave it like that but try to not push code that you're not currently using